nav2_bringup/config/control.yaml

```diff
@@ -0,0 +1,78 @@
+/**:
+  controller_manager:
+    ros__parameters:
+      update_rate: 1000  # Hz
+
+      joint_state_broadcaster:
+        type: joint_state_broadcaster/JointStateBroadcaster
+
+      diffdrive_controller:
+        type: diff_drive_controller/DiffDriveController
+
+  diffdrive_controller:
```

Comment:
I'd put this file in `urdf` since having both a `config` and a `params` directory seems confusing + this is strictly related code to the URDF